protocol/chainlib/rest_test.go

```diff
@@ -111,8 +111,9 @@ func TestRestParseMessage(t *testing.T) {
 	assert.Equal(t, msg.GetServiceApi().Name, apip.serverApis["API1"].Name)
 
 	restMessage := rpcInterfaceMessages.RestMessage{
-		Msg:  []byte("test message"),
-		Path: "API1",
+		Msg:      []byte("test message"),
+		Path:     "API1",
+		SpecPath: "API1",
```

Comment:
if Spec path is not used in this test you don't need to specify it, it will just be empty by default.

Reply:
It is actually used in the unit tests because `ParseMsg()` assigns `SpechPath` to `msg` parameter, so it needs to be specified here as well.